src/linalg/cholesky.rs

```diff
@@ -74,6 +74,13 @@ where
         Cholesky { chol: matrix }
     }
 
+    /// Uses the given matrix as-is without any checks or modifications as the
+    /// Cholesky decomposition. It is up to the user to ensure and all
+    /// invariants hold.
```

Comment:
Very minor, but maybe put the second sentence in a new paragraph, since the first line is used as a "summary" in the docs. Also, the double "and" is removed, but I think maybe the "and" here also shouldn't be there...? That is, shouldn't it say "to ensure all invariants hold"?

Reply:
You're right, done.